lib/cc/analyzer/engine.rb

```diff
@@ -34,7 +34,7 @@ def run(stdout_io, container_listener)
         )
 
         container.on_output("\0") do |raw_output|
-          CLI.debug "#{name} engine output: #{raw_output.inspect}"
+          CLI.debug("#{name} engine output: #{raw_output.gsub("\n", "\\n").gsub("\t", "\\t")}")
```

Comment:
> WDYT of gsub(/[\n\t]/, " ")?

Could do, but I'm hesitant to eliminate information about the *kind* of whitespace present in something calling itself a debug mode. That whitespace could also be present *within* the issue, where it's kind may be more important to interpreting the results.

> There's also JSON.pretty_generate but I think you want it to all be one line?

We definitely want this on one line. Also, this debug output is before anything else has tried to parse the output has JSON, I believe. I wouldn't want invalid-engine-output to start raising from here instead of from where we want it to.